tests/test_multiple_inheritance.cpp

```diff
@@ -193,14 +193,12 @@ TEST_SUBMODULE(multiple_inheritance, m) {
         .def_readwrite_static("static_value", &VanillaStaticMix2::static_value);
 
 
-#if !(defined(PYPY_VERSION) && (PYPY_VERSION_NUM < 0x06000000))
```

Comment:
Huh? If this was meant for PYPY < 6, then shouldn't this be thrown out, instead of allowed on 7?

Reply:
Oops, misread that. Thanks!

Reply:
Actually, I misread this.

`NOT(PYPY AND PYPY < 6)` would be `NOT PYPY OR PYPY >= 6` which would mean either CPython or PyPy 6+. Since we dropped support for PyPy < 7, your original commit was right.

a) Sorry
b) Feel free to drop the last commit.

Reply:
Undone. :)